.github/workflows/workflow-build-and-deploy-scaleway.yml

```diff
@@ -70,6 +129,14 @@ jobs:
           github-access-token: ${{ secrets.GITHUB_TOKEN }}
           tag: ${{ inputs.environment }}
           platforms: ${{ vars.CONTAINER_PLATFORMS }}
+
+  k8s-deploy:
```

Comment:
Hier misschien ook een matrix van maken wanneer het een monorepo is, zodat we echt het project apart kunnen herstarten, momenteel herstart hij alle ingestelde $K8S_LABELS

Reply:
The deploy step should be part of the build above (once it's back to 1 job for everything). Then we can just have a step that deploys to the right variant.

We namely have an image suffix, such as `-myVariant` (or empty for normal repo). We could append that to the labels.

But it's an improvement for later 🤷